package.json

```diff
@@ -21,7 +21,7 @@
     "start": "craco start",
     "build": "craco build",
     "serve": "serve build -l 3000",
-    "lint": "yarn eslint --ignore-path .gitignore .",
+    "lint": "yarn eslint --ignore-path .gitignore --cache --cache-location node_modules/.cache/eslint/ .",
```

Comment:
should lint/type cache files live outside node_modules?

Reply:
`node_modules/.cache` is a community-driven standard for where to put tooling-related caches - see [find-cache-dir](https://github.com/sindresorhus/find-cache-dir).
We already use this, it's just not explicit yet. For example, if you log the eslint config from react-scripts, it is caching in `node_modules/.cache/.eslintcache`; if you check `node_modules/.cache`, you'll also see babel and terser caching there.
I don't want to use a new arbitrary folder for doing so, and have to manage upkeep for that as well (ie `.gitignore`) - I'd rather use an existing standard.